src/views/dialogs/LaunchMarketDialog.tsx

```diff
@@ -86,7 +89,7 @@ export const LaunchMarketDialog = ({ setIsOpen }: DialogProps<LaunchMarketDialog
       setIsOpen={setIsOpen}
       placement={isMobile ? DialogPlacement.FullScreen : DialogPlacement.Default}
     >
-      <NewMarketForm setFormStep={setFormStep} />
+      <NewMarketForm launchableMarketId={launchableMarketId} setFormStep={setFormStep} />
```

Comment:
technically this is just the default one right? makes more sense to have this prop be called `defaultLaunchableMarketId` or something

Reply:
Sounds good!